src/Sylius/Behat/spec/Context/Ui/Admin/ManagingLocalesContextSpec.php

```diff
@@ -101,4 +101,11 @@ function it_throws_an_exception_if_resource_can_not_be_founded_on_page(IndexPage
 
         $this->shouldThrow(\InvalidArgumentException::class)->during('storeShouldBeAvailableInLanguage', ['Norwegian']);
     }
+
+    function it_throws_exception_if_locale_name_cannot_be_chosen_again(CreatePageInterface $createPage)
+    {
+        $createPage->chooseName('France')->willThrow(\Exception::class);
+
+        $this->shouldThrow(\Exception::class)->during('iShouldNotBeAbleToChoose', ['France']);
```

Comment:
`chooseName()` method won't throw exception cuz in spec we test our object isolated from UI so it doesn't know about presentation layer.